test/downgrade/doesnt_change_git_dependencies_test.dart

```diff
@@ -18,23 +18,15 @@ main() {
       "foo": {"git": "../foo.git"}
     }).create();
 
-    // TODO(rnystrom): Remove "--packages-dir" and validate using the
-    // ".packages" file instead of looking in the "packages" directory.
-    await pubGet(args: ["--packages-dir"]);
+    await pubGet();
 
-    await d.dir(packagesPath, [
-      d.dir('foo', [d.file('foo.dart', 'main() => "foo";')])
-    ]).validate();
+    final originalFooSpec = packageSpecLine('foo');
```

Comment:
Nit: We don't use "final" for local variables in pub even when we could. I don't think it adds much benefit in general, and I think it's good to be consistent.

Reply:
Done.